### ISSUE TITLE
orientdb issues on nixos

### ISSUE THREAD

**Issue by moaxcp** [jbake-org/jbake#463]
I started getting this exception after an upgrade

```
Caused by: java.lang.NoSuchMethodError: com.orientechnologies.common.jna.OCLibrary$Rlimit.createFieldsOrder([Ljava/lang/String;)Ljava/util/List;
    	at com.orientechnologies.common.jna.OCLibrary$Rlimit.<clinit>(OCLibrary.java:21)
    	at com.sun.proxy.$Proxy81.<clinit>(Unknown Source)
    	at com.sun.jna.Native.loadLibrary(Native.java:415)
    	at com.sun.jna.Native.loadLibrary(Native.java:391)
    	at com.orientechnologies.common.jna.ONative.instance(ONative.java:31)
    	at com.orientechnologies.orient.core.engine.OMemoryAndLocalPaginatedEnginesInitializer.configureDefaultDiskCacheSize(OMemoryAndLocalPaginatedEnginesInitializer.java:167)
    	at com.orientechnologies.orient.core.engine.OMemoryAndLocalPaginatedEnginesInitializer.configureDefaults(OMemoryAndLocalPaginatedEnginesInitializer.java:77)
    	at com.orientechnologies.orient.core.engine.OMemoryAndLocalPaginatedEnginesInitializer.initialize(OMemoryAndLocalPaginatedEnginesInitializer.java:62)
    	at com.orientechnologies.orient.core.engine.memory.OEngineMemory.startup(OEngineMemory.java:62)
    	at com.orientechnologies.orient.core.Orient.startEngine(Orient.java:930)
    	at com.orientechnologies.orient.core.Orient.loadStorage(Orient.java:523)
    	at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.<init>(ODatabaseDocumentTx.java:173)
    	at com.orientechnologies.orient.core.db.OPartitionedDatabasePool$DatabaseDocumentTxPooled.<init>(OPartitionedDatabasePool.java:428)
    	at com.orientechnologies.orient.core.db.OPartitionedDatabasePool$DatabaseDocumentTxPooled.<init>(OPartitionedDatabasePool.java:424)
    	at com.orientechnologies.orient.core.db.OPartitionedDatabasePool.initQueue(OPartitionedDatabasePool.java:358)
    	at com.orientechnologies.orient.core.db.OPartitionedDatabasePool.<init>(OPartitionedDatabasePool.java:117)
    	at com.orientechnologies.orient.core.db.OPartitionedDatabasePoolFactory.get(OPartitionedDatabasePoolFactory.java:116)
    	at org.jbake.app.ContentStore.<init>(ContentStore.java:61)
    	at org.jbake.app.DBUtil.createDataStore(DBUtil.java:15)
    	at org.jbake.app.Oven.bake(Oven.java:132)
    	at org.jbake.app.Oven$bake$0.call(Unknown Source)
    	at org.jbake.gradle.JBakeProxyImpl.jbake(JBakeProxyImpl.groovy:33)
    	at org.jbake.gradle.JBakeProxy$jbake$2.call(Unknown Source)
    	at org.jbake.gradle.JBakeTask.bake(JBakeTask.groovy:46)
    	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:73)
    	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.doExecute(StandardTaskAction.java:46)
    	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:39)
    	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:26)
    	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:784)
    	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:751)
    	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.run(ExecuteActionsTaskExecuter.java:121)
    	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
    	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
    	at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:199)
    	at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:110)
    	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:110)
    	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:92)
    	... 29 more
```

It looks like I am running into orientechnologies/orientdb#7911. I'm going to try upgrading the version used by the gradle plugin if possible.

**Comment by moaxcp** [comment]
```
buildscript {
    repositories {
        jcenter()
    }
    dependencies {
        classpath 'com.orientechnologies:orientdb-core:2.2.34'
    }
}
```
fixed the issue. This is for a gradle project using the org.jbake.site plugin.

**Comment by ancho** [comment]
oh. interesting. thanks for catching this. if you want you can open a PR to update the version of orientdb for jbake-core. In the meantime orient published 2.2.35.

**Comment by ancho** [comment]
Should be fixed with merge of #494
